helm/coredns-app/templates/_helpers.tpl

```diff
@@ -26,3 +26,15 @@ Selector labels
 {{- define "labels.selector" -}}
 k8s-app: {{ .Values.name | quote }}
 {{- end -}}
+
+{{- define "cloudprovider.forward" -}}
+{{- if hasKey .Values.cloudSpecificSettings .Values.provider -}}
+{{- $cloudProviderConfig := get .Values.cloudSpecificSettings .Values.provider}}
+  {{- $zones := get $cloudProviderConfig "defaultZones" }}
+  {{- $zones = concat $zones .Values.cloudSpecificSettings.additionalZones}}
+  {{- $global := . }}
```

Comment:
Where is this used?

Reply:
Leftover.
Please note this is a draft pr being worked on heavily